pwa/src/components/entities/entityForm.tsx

```diff
@@ -147,17 +149,27 @@ export const EntityForm: React.FC<EntityFormProps> = ({ entityId }) => {
                           name="function"
                           label="Function"
                           options={functionSelectOptions}
-                          validation={{ required: true }}
+                          validation={{ validate: () => validateLength(getValues("name")), required: true }}
```

Comment:
The `validation` prop can hold multiple properties. Right now, we're using `required` and you've just added `validate`.

However, you're not writing a custom validation function for a `maxLength` validation, `maxLength` is a predefined validation that `react-hook-form` offers (the same as `required`).

You can add this like: `maxLength: 255` (in case the max length is 255).

This would result in:

```
validation={{ maxLength: 255, required: true }}
```